test/e2e/e2e_test.go

```diff
@@ -0,0 +1,201 @@
+			},
+		},
+		{
+			Summary:  "Pulling image \"nginx\"",
+			Category: "kubernetes",
+			Attributes: map[string]interface{}{
+				"event.metadata.managedFields[0].operation": "Update",
+				"event.metadata.name":                       "nginx-e2e.",
+				"event.metadata.namespace":                  ns,
+				"event.reason":                              "Pulling",
+				"clusterName":                               "",
+				"event.involvedObject.apiVersion":           "",
+				"event.involvedObject.kind":                 "Pod",
+				"event.involvedObject.name":                 "nginx-e2e",
+				"event.message":                             "Pulling image \"nginx\"",
+				"verb":                                      "ADDED",
+			},
+		},
+		{
+			Summary:  "Successfully pulled image \"nginx\" in ",
+			Category: "kubernetes",
+			Attributes: map[string]interface{}{
+				"event.metadata.managedFields[0].operation": "Update",
+				"event.metadata.name":                       "nginx-e2e.",
+				"event.metadata.namespace":                  ns,
+				"event.reason":                              "Pulled",
+				"clusterName":                               "",
+				"event.involvedObject.apiVersion":           "",
+				"event.involvedObject.kind":                 "Pod",
+				"event.involvedObject.name":                 "nginx-e2e",
+				"event.message":                             "Successfully pulled image \"nginx\" in ",
+				"verb":                                      "ADDED",
+			},
+		},
+		{
+			Summary:  "Created container nginx",
+			Category: "kubernetes",
+			Attributes: map[string]interface{}{
+				"event.metadata.managedFields[0].operation": "Update",
+				"event.metadata.name":                       "nginx-e2e.",
+				"event.metadata.namespace":                  ns,
+				"event.reason":                              "Created",
+				"clusterName":                               "",
+				"event.involvedObject.apiVersion":           "",
+				"event.involvedObject.kind":                 "Pod",
+				"event.involvedObject.name":                 "nginx-e2e",
+				"event.message":                             "Created container nginx",
+				"verb":                                      "ADDED",
+			},
+		},
+		{
+			Summary:  "Started container nginx",
+			Category: "kubernetes",
+			Attributes: map[string]interface{}{
+				"event.metadata.managedFields[0].operation": "Update",
+				"event.metadata.name":                       "nginx-e2e.",
+				"event.metadata.namespace":                  ns,
+				"event.reason":                              "Started",
+				"clusterName":                               "",
+				"event.involvedObject.apiVersion":           "",
+				"event.involvedObject.kind":                 "Pod",
+				"event.involvedObject.name":                 "nginx-e2e",
+				"event.message":                             "Started container nginx",
+				"verb":                                      "ADDED",
+			},
+		},
+	} {
+		if !agentMock.Has(&event) {
```

Comment:
So you can shift the indentation?
```suggestion
		if agentMock.Has(&event) {
		  return
		}
```